### PR TITLE
remove tutorial and update link in help text

### PR DESCRIPTION
**Pull request by kingsleyb**
PR Checklist:

- [x] add a short description of what's changed to the top of the `CHANGELOG.md`
- [ ] add/update tests (or don't, for reasons explained below)

## Describe this PR
This specific tutorial that was focussed on demonstrating cloud features in the UI rather than running code locally and registering with cloud does not fit the new markdown model tutorial. Given that changes to this tutorial would end up being duplicative of the existing universal deploy tutorial the decision was made to remove this.

I also updated the links in the agents page to navigate to this tutorial page.

### CODE REVIEW

File: src/components/Agents/Agents.vue [hunk 1]
```diff
@@ -431,11 +431,7 @@ export default {
...
           You can also go through the
-          <router-link class="link" :to="{ name: 'flow-run-tutorial' }">
-            <u>Running a Flow</u>
-          </router-link>
-          and
-          <router-link class="link" :to="{ name: 'universal-deploy-tutorial' }">
+          <router-link class="link" :to="{ name: 'tutorial' }">
             <u>Universal Deploy</u>
           </router-link>
           tutorials for guidance on running agents in Cloud.</span
```
Comment (zhen0) [inline src/components/Agents/Agents.vue:437]:
```suggestion
          tutorial for guidance on running agents in Cloud.</span
```

**Review by zhen0 — COMMENTED** [review]
One very small suggestion from me @kingsleyb but otherwise looks good.  Definitely in agreement that we should take this out.

File: src/components/Agents/Agents.vue (hunk 1 above)
Comment (kingsleyb) [reply src/components/Agents/Agents.vue]: ah yes, nice catch, thank you

**Review by znicholasbrown — APPROVED** [review]
LGTM @kingsleyb ! 👍

**Pull request merged**